build: remove `fullTemplateTypeCheck` from aio tsconfig

alan-agius4 opened the pull request:
`fullTemplateTypeCheck` is no longer required since we now use `strictTemplates` which is a superset of the former option.

---

Follow up on https://github.com/angular/angular/commit/04f61c0c3ee9b3616b1719b4ae9c3eb097698218#r38354112

//cc @IgorMinar

gkalpak reviewed (approved):
✨

Commit message typo: `requires` --> `required`
(Also, it would be useful to mention this is a follow-up to https://github.com/angular/angular/commit/04f61c0c3ee9b3616b1719b4ae9c3eb097698218#r38354112 in the commit message.)

alan-agius4 commented:
@gkalpak commit message updated.

alan-agius4 committed:
build: remove `fullTemplateTypeCheck` from aio tsconfig

`fullTemplateTypeCheck` is no longer required since we now use `strictTemplates` which is a superset of the former option.

Follow-up on: https://github.com/angular/angular/commit/04f61c0c3ee9b3616b1719b4ae9c3eb097698218#r38354112
